main.go

```diff
@@ -59,12 +60,16 @@ func main() {
 		KeyFilePath:         keyFilePath,
 	}
 
+	eventDispatcher := event.NewEventDispatcher()
+	defer eventDispatcher.Close()
```

Comment:
In use case.

Reply:
I didn't get here. How are you going to move it in `use case`?

Reply:
@alldroll Sorry, I misunderstood this one. Please ignore my previous comment.